cassandane/data/directory.ldif

```diff
@@ -38,12 +38,16 @@ objectclass: group
 cn: group co
 ou: groups
 o: cyrus
+hasmember: cassandane
+hasmember: otheruser
```

Comment:
Usually these would be dns, not just uids, because the directory wants to be able to enforce foreign key constraints and so on.  Compare for example the memberof attribute within the user objects, whose values are the dns of the groups.  Also, I'd expect this attribute to typically be called just "member" too, or perhaps be named to denote a specific kind of member (real world examples from memory: "uniquemember", "rfc822mailmember", etc), but not "hasmember".

We can make directory.ldif say whatever the hell we want, but it's important to remember that if it isn't representative of a plausible real world directory layout, then testing against it doesn't prove anything about whether Cyrus's LDAP support works, and a test that doesn't prove anything is a waste of time